data/plugins/skill/mining/mining.rb

```diff
@@ -21,12 +21,7 @@ def initialize(mob, position, ore)
   end
 
   def find_pickaxe
-    weapon = mob.equipment.get(EquipmentConstants::WEAPON)
-    PICKAXE_IDS.each do |id|
-      return PICKAXES[id] if (!weapon.nil? && weapon.id == id) || mob.inventory.contains(id)
-    end
-
-    nil
+    PICKAXES[PICKAXE_IDS.select {|id| PICKAXES[id].level <= mob.skill_set.get_skill(Skill::MINING).current_level && (mob.inventory.contains(id) || mob.equipment.contains(id))  }.max]
```

Comment:
Could you break this condition down into multiple lines? It spans too many columns.